modules/porous_flow/doc/content/source/dirackernels/PorousFlowPointSourceFromPostprocessor.md

```diff
@@ -0,0 +1,18 @@
+# PorousFlowPointSourceFromPostprocessor
+
+`PorousFlowPointSourceFromPostprocessor`
+implements a mass point source that adds (removes) fluid at a
+mass flux rate which was computed by a postprocessor.
+
+For instance:
+
+!listing modules/porous_flow/test/tests/dirackernels/frompps.i block=DiracKernels
+
+Note that the ```execute_on``` parameter is set to ```timestep_begin``` so that the correct
+value is being used with the timestep.
```

Comment:
You should show the Postprocessor block so readers know what you're talking about with the `timestep_begin` stuff

Reply:
Good idea! 💡

Reply:
And I should say "with**in** the timestep"